packages/blob/README.md

```diff
@@ -56,33 +175,15 @@ async function put(
 
 ### del(url, options)
 
-Delete one or multiple blobs by their full URL. Returns the deleted blob(s) or null when not found.
+Delete one or multiple blobs by their full URL.
 
 ```ts
 async function del(
   url: string | string[],
   options?: {
     token?: string;
   },
-): Promise<
-  | {
-      size: number;
-      uploadedAt: Date;
-      pathname: string;
-      contentType: string;
-      contentDisposition: string;
-      url: string;
-    }
-  | null
-  | ({
-      size: number;
-      uploadedAt: Date;
-      pathname: string;
-      contentType: string;
-      contentDisposition: string;
-      url: string;
-    } | null)[]
-> {}
+): Promise<string | string[]> {}
```

Comment:
just wondering if we want to keep the `{ url: string }` interface, which is extendable and more compatible with the previous interface

Reply:
👍 for { url: string } and compatibility, but I am even wondering if we should return anything, the R2 API sends back "void" (https://developers.cloudflare.com/r2/api/workers/workers-api-reference/) and that may be the good choice. Because S3 sends back some data, but it's useless. So if the return value of delete is always useless, maybe `void` is the right thing.

Reply:
as we support multiple deletes, it may be useful to know which files have been deleted and which threw an error (I don't think we are handling this case in the worker though)

Reply:
> which files have been deleted

Neither R2 nor S3 can tell us about that unfortunately. They just say "If you ask for it to be deleted, it is deleted if it exists". i.e. if you want to know if it's deleted, you only need to know if it existed before.

That's why they probably decided to return void. I am inclined to do the same as then the API response is very clear: no you can't know if it worked: it worked if the files were here

Reply:
ok, then the only reason to keep it would be for compatibility with the old api. I'd be fine either to return either `{url : string}` or just `void`

Reply:
> then the only reason to keep it would be for compatibility with the old api

I'd rather optimize for new users rather than previous users of the module (because our biggest opportunity are new users). I will send void and update the Vercel Blob API too now

Reply:
now sending `null` always

Reply:
can't we just skip the return and set `Promise<void`> as return type?

Reply:
I wanted to do that but then you can't have a type DelBlobResult which is "void", so I thought "perhaps this is not a good idea and null is 'better'". Thoughts?

Reply:
<img width="1618" alt="CleanShot 2023-07-18 at 1 05 51" src="https://github.com/vercel/storage/assets/123822/ae598baf-df59-48da-9563-945d6d82bd2a">

hmm that's eslint maybe we can ignore this

Reply:
we can remove this type, the user wouldn't need it anymore

Reply:
done and now returning void